routes/users.js

```diff
@@ -14,4 +15,16 @@ router.post("/candlestick", async function (req, res, next) {
   }
 });
 
+router.get("/asset/:userid", verifyToken, async function (req, res, next) {
+  const { userId } = req.params;
+
+  try {
+    const user = await User.findById({ _id: userId }).lean().exec();
```

Comment:
```suggestion
    const user = await User.findById(userId).lean().exec();
```
이렇게도 작성할 수 있을 거 같아요 ~ 지금도 좋습니다 !